### PR TITLE
updateMaterial and angularCLI

### PR DESCRIPTION
**Pull request by alan345**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,16 +14,19 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "^4.1.0",
+    "@angular/cdk": "github:angular/cdk-builds",
```
Comment (alan345) [inline package.json]: new cdk-builds from material: https://material.angular.io/guide/getting-started

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,21 +1,22 @@
 <toaster-container></toaster-container>
 
 <div fxLayout="column" fxFlex>
-  <md-toolbar color="primary">
+  <mat-toolbar color="primary">
```
Comment (alan345) [inline src/app/app.component.html]: rename all "md" to "mat"

**Comment by dassiorleando** [comment]
Ok cool @alan345, I'll merge it.

File: package.json (hunk 1 above)
Comment (dassiorleando) [reply package.json]: That looks cool.

**Comment by alan345** [comment]
Cool! Test it as it might have still some bugs

**Comment by dassiorleando** [comment]
@alan345 it breaks the original design of the starter, I just merge it into the develop branch for the moment once it's Ok we merge with the master branch.

**Pull request merged**